p2p/security/noise/session.go

```diff
@@ -36,22 +36,24 @@ type secureSession struct {
 	dec *noise.CipherState
 
 	// noise prologue
-	prologue         []byte
-	earlyDataHandler EarlyDataHandler
+	prologue []byte
+
+	initiatorEarlyDataHandler, receiverEarlyDataHandler EarlyDataHandler
 }
```

Comment:
same suggestion for calling this `responder`

Reply:
Done.